package.json

```diff
@@ -28,43 +28,59 @@
   },
   "homepage": "https://github.com/tensorflow/tensorboard#readme",
   "devDependencies": {
-    "@angular/bazel": "^8.2.14",
-    "@angular/cli": "^8.3.21",
-    "@angular/compiler": "^8.2.14",
-    "@angular/compiler-cli": "^8.2.14",
-    "@bazel/bazel": "0.26.1",
-    "@bazel/hide-bazel-files": "^0.34.0",
+    "@angular/bazel": "^9.0.0",
+    "@angular/cli": "^9.0.1",
+    "@angular/compiler": "^9.0.0",
+    "@angular/compiler-cli": "^9.0.0",
+    "@bazel/bazel": "^0.23.2",
```

Comment:
This actually downgrades our `@bazel/bazel` from `0.26.1` to just `0.23.2` (per the impact on yarn.lock).  Was that actually necessary?

If not, it seems better if we can to use 2.1.0 to match our Bazel version, since it looks like this package is released approximately on the overall Bazel release schedule:
https://www.npmjs.com/package/@bazel/bazel/v/2.1.0?activeTab=versions